### PR TITLE
Fix typos and add codespell GitHub Action

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S .mypy_cache`

### CODE REVIEW

**Review by balamurugana — CHANGES_REQUESTED** [review]
Could you add `codespell` into github workflow as well?

**Comment by kianmeng** [comment]
@balamurugana As a separate individual workflow or append into existing one in https://github.com/minio/minio-py/tree/master/.github/workflows?

**Comment by balamurugana** [comment]
> @balamurugana As a separate individual workflow or append into existing one in https://github.com/minio/minio-py/tree/master/.github/workflows?

Just add it to https://github.com/minio/minio-py/blob/master/.github/workflows/pythonpackage-linux.yml

File: .github/workflows/pythonpackage-linux.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
         os: [ubuntu-latest]
 
     steps:
+    - uses: codespell-project/actions-codespell@master
```
Comment (balamurugana) [inline .github/workflows/pythonpackage-linux.yml:22]: It is not working

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: .github/workflows/pythonpackage-linux.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - uses: codespell-project/actions-codespell@master
```
Comment (balamurugana) [inline .github/workflows/pythonpackage-linux.yml:23]: You have to `pip install` and run the checker manually. Try enabling github workflow in your fork of minio-py and test it before refreshing this PR.

**Review by balamurugana — CHANGES_REQUESTED** [review]

File: .github/workflows/pythonpackage-linux.yml (hunk 2 above)
Comment (kianmeng) [reply .github/workflows/pythonpackage-linux.yml:23]: Seems to work, but it was enabled for all Python versions.

image

 Maybe we can add this as a separate workflow instead?

**Review by balamurugana — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**